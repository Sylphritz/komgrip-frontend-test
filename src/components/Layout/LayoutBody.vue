<template>
  <div class="container-fluid py-4">
    <slot />
    <LayoutFooter />
  </div>
</template>

<script setup lang="ts">
import LayoutFooter from './LayoutFooter.vue'
</script>
