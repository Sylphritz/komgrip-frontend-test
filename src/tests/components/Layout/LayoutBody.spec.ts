import { describe, it, expect } from 'vitest'

import { mount } from '@vue/test-utils'
import LayoutBody from '@/components/Layout/LayoutBody.vue'

describe('LayoutBody', () => {
  it('renders successfully and matches the snapshot', () => {
    const wrapper = mount(LayoutBody, {
      shallow: true
    })

    expect(wrapper.element).toMatchSnapshot()
  })
})
