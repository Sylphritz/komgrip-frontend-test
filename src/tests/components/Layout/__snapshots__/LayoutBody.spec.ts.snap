// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LayoutBody > renders successfully and matches the snapshot 1`] = `
<div
  class="container-fluid py-4"
>
  
  
  <layout-footer-stub />
</div>
`;
